### PR TITLE
Add an option to inject additional objects to load config

### PR DESCRIPTION
**Pull request by alan-kut**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
This PR allows to inject additional objects via CL2_ADDITIONAL_OBJECTS_MODULES to load/config.yaml.

### CODE REVIEW

File: clusterloader2/testing/load/config.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@
 {{$defaultQps := DefaultParam .CL2_DEFAULT_QPS (IfThenElse (le .Nodes 500) 10 100)}}
 
 {{$ADDITIONAL_MEASUREMENT_MODULES := DefaultParam .CL2_ADDITIONAL_MEASURMENT_MODULES nil}}
+{{$ADDITIONAL_OBJECTS_MODULES := DefaultParam .CL2_ADDITIONAL_OBJECTS_MODULES nil}}
```
Comment (tosi3k) [inline clusterloader2/testing/load/config.yaml:81]: Could we rename this to `ADDITIONAL_PHASES_MODULES` and `CL2_ADDITIONAL_PHASES_MODULES`?

This sounds more in line with the CL2's API where a single step can be either a list of measurements or phases.

Comment (alan-kut) [reply clusterloader2/testing/load/config.yaml]: Done

File: clusterloader2/testing/load/config.yaml [hunk 2]
```diff
@@ -207,6 +208,15 @@ steps:
       smallJobSize: {{$SMALL_GROUP_SIZE}}
       smallJobsPerNamespace: 1
 
+{{if $ADDITIONAL_PHASES_MODULES}}
```
Comment (mborsz) [inline clusterloader2/testing/load/config.yaml:211]: Nit: is this if necessary? Range over nil should be legal operation.

Comment (alan-kut) [reply clusterloader2/testing/load/config.yaml:211]: I followed what was done in the https://github.com/kubernetes/perf-tests/pull/2483

**Pull request merged**